onnxscript/rewriter/fuse_relus_clips_test.py

```diff
@@ -0,0 +1,320 @@
+                Y = Relu(x2)
+            }
+        """)
+        self.run_test(model, expected_op_type="Relu")
+
+
+class FuseSuccessiveReluClipTest(_FuseReluClipTestBase):
+    @parameterized.parameterized.expand(
+        [
+            (
+                "relu_then_clip",
+                """
+                    x1 = Relu(X)
+                    Y = Clip(x1, min, max)
+                """,
+                "float",
+            ),
+            (
+                "clip_then_relu",
+                """
+                    x1 = Clip(X, min, max)
+                    Y = Relu(x1)
+                """,
+                "float",
+            ),
+            (
+                "int_relu_then_clip",
+                """
+                    x1 = Relu(X)
+                    Y = Clip(x1, min, max)
+                """,
+                "int32",
+            ),
+            (
+                "int_clip_then_relu",
+                """
+                    x1 = Clip(X, min, max)
+                    Y = Relu(x1)
+                """,
+                "int32",
+            ),
+        ]
+    )
+    def test_fuse_successive_relu_clip(self, _, nodes, dtype):
+        model = ir.from_onnx_text(f"""
+            < ir_version: 10, opset_import: ["" : 20] >
+            test_model ({dtype}[N, 32, 14] X) => ({dtype} [N, ?, ?] Y)
+            <{dtype} min = {{1}}, {dtype} max = {{6}}>
+            {{
+                {nodes}
+            }}
+        """)
+        self.run_test(model, expected_op_type="Clip", dtype=dtype)
+
+    @parameterized.parameterized.expand(
+        [
+            (
+                "relu_then_clip",
+                """
+                    x1 = Relu(X)
+                    Y = Clip(x1,,max)
+                """,
+            ),
+            (
+                "clip_then_relu",
+                """
+                    x1 = Clip(X,,max)
+                    Y = Relu(x1)
+                """,
+            ),
+        ]
+    )
+    def test_fuse_successive_relu_clip_no_min(self, _, nodes):
+        model = ir.from_onnx_text(f"""
+            < ir_version: 10, opset_import: ["" : 20] >
+            test_model (float[N, 32, 14] X) => (float [N, ?, ?] Y)
+            <float max = {{6.0}}>
+            {{
+                {nodes}
+            }}
+        """)
+        self.run_test(model, expected_op_type="Clip")
+
+    @parameterized.parameterized.expand(
+        [
+            (
+                "relu_then_clip",
+                """
+                    x1 = Relu(X)
+                    Y = Clip(x1, min)
+                """,
+                fuse_successive_clip_relu_rule,
+            ),
+            (
+                "clip_then_relu",
+                """
+                    x1 = Clip(X, min)
+                    Y = Relu(x1)
+                """,
+                fuse_successive_relu_clip_rule,
+            ),
+        ]
+    )
+    def test_fuse_successive_relu_clip_non_initializers(self, _, nodes, rewrite_rule):
```

Comment:
nit: Could you add "successful" or "fail" on the test title so people can know what to expect?